markus_test_scripts/examples/test_markus_integration.R

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env Rscript
+
+# Load helper functions
+source("r_helpers.R")
+
+# Test basic LLM function with a simple example
+test_basic_functionality <- function() {
+  test_code <- 'x <- 1 + 1\nprint(x)'
+  writeLines(test_code, "test_submission.R")
```

Comment:
The test should not be generating a fake student file. The student file should be stored as a separate file (it's what the student will upload to MarkUs; see my overall review comment). The student file should be called something like `submission.R`; it doesn't make sense for it to have a "test" name.